### ISSUE TITLE
Create `<release.>yaml`

### ISSUE THREAD

**Issue by dingp** [DUNE-DAQ/proto-spack#9]
Create release specific yaml file, e.g. `develop.yaml`, `N22-03-23.yaml`, `dunedaq-v2.10.0.yaml` with lists of DAQ packages and externals. 

These files live under `proto-spack/releases`. `proto-spack/dune-build/externals/package.py`, `proto-spack/dune_daqpackages/<pkg>/package.py` will get information from it, including release name, package list, version, commit hash etc. 

A proposed format of the file looks like: 

```yaml=
release: N22-03-23:
packages:
  - name: appfwk
    version: "hash"
    variant: ~
  ## list continues
externals:
  - name: gcc
    version: "v8_2_0"
    variant: ~
  ## list continues
```

**Comment by dingp** [comment]
Implemented in PR DUNE-DAQ/daq-release#128.

**Comment by dingp** [comment]
Done. PR https://github.com/DUNE-DAQ/daq-release/pull/128 has been merged.
